src/common/components/DropdownMenu/DropdownMenu.jsx

```diff
@@ -54,20 +49,25 @@ class DropdownMenu extends Component {
 
 DropdownMenu.propTypes = {
   items: PropTypes.arrayOf(
-    PropTypes.shape({
-      onClick: PropTypes.func,
-      href: PropTypes.string,
-      to: PropTypes.string,
-      display: PropTypes.string.isRequired,
-      target: PropTypes.string,
-    })
+    PropTypes.oneOfType([
+      PropTypes.shape({
+        onClick: PropTypes.func,
+        href: PropTypes.string,
+        to: PropTypes.string,
+        display: PropTypes.string.isRequired,
+        target: PropTypes.string,
+      }),
+      PropTypes.node,
+    ])
   ).isRequired,
   title: PropTypes.node.isRequired,
   titleClassName: PropTypes.string,
+  dataTestId: PropTypes.string,
```

Comment:
is this really necessary?

Reply:
adding props just for the tests seems weird

Reply:
not necessary if you're willing to have classname or some other kind of selector, which will result in mess in the future. 

Also I think adding a prop for testability is not a bad thing.

Reply:
maybe it's better to rename it as a generic ``id`` prop instead of  a``test`` ``id``